app/assets/stylesheets/helpers/_webchat.scss

```diff
@@ -0,0 +1,35 @@
+.webchat-banner {
+  background-color: $govuk-blue;
+  color: $white;
+  @include core-19;
+  max-height: 0;
+  padding: 0 $gutter-half;
+  opacity: 0;
+  margin-bottom: 0;
+  -webkit-transition: .7s ease-out;
+  -webkit-transition-property: max-height, opacity, margin-bottom, padding-top, padding-bottom;
+  transition: .7s ease-out;
+  transition-property: max-height, opacity, margin-bottom, padding-top, padding-bottom;
```

Comment:
We have a [transition helper mixin](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_css3.scss#L57) in the FET